.orchestra/support/save-for-later

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+from tarfile import open as tar_open
+from typing import NoReturn
+
+INSTALL_DIRECTORY = f"{os.environ['DESTDIR']}{os.environ['ORCHESTRA_ROOT']}"
+ORCHESTRA_SHARE = os.path.join(INSTALL_DIRECTORY, "share/orchestra")
+SAVE_DIRECTORY = os.path.join(ORCHESTRA_SHARE, "save_for_later")
+
+
+def fail(msg: str) -> NoReturn:
+    sys.stderr.write(f"{msg}\n")
+    sys.exit(1)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", type=str, help="Input Directory")
+    parser.add_argument("name", type=str)
+    parser.add_argument("pattern", type=str, nargs="+")
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    patterns = [re.compile(p) for p in args.pattern]
+
+    if args.input is not None:
+        if not os.path.isdir(args.input):
+            fail("Input directory does not exist")
+        os.chdir(args.input)
+
+    selected_files = []
+    for root, _, files in os.walk("."):
+        for file in files:
+            full_file = os.path.join(root, file)[2:]
+            if any(p.match(full_file) for p in patterns):
+                selected_files.append(full_file)
+
+    if not os.path.isdir(SAVE_DIRECTORY):
```

Comment:
I thought you were a `Pathlib` fag.

Reply:
If it's a one-off thing I'll use `os.path`, but you're right, I should switch to `pathlib`